samples/tableadmin.js

```diff
@@ -13,293 +13,381 @@
 // limitations under the License.
```

Comment:
For pretty much all the samples in this section we are pulling:

```
const {Bigtable, GCRuleMaker} = require('@google-cloud/bigtable');
const {BigtableTableAdminClient} = require('@google-cloud/bigtable').v2;
const adminClient = new BigtableTableAdminClient();
```

into the samples and indenting the code that contains a sample so that these declarations can be done multiple times.